test/test_integration_systemd.rb

```diff
@@ -0,0 +1,64 @@
+require_relative "helper"
+require_relative "helpers/integration"
+
+require 'sd_notify'
+
+class TestIntegrationSystemd < TestIntegration
+  def setup
+    skip "Skipped on Windows because it does not support Systemd" if windows?
```

Comment:
Hm, shouldn't this test just be Linux-only, really?

Reply:
Can you give me more clarification on that?

Reply:
@acmh systemd only exist for Linux

Reply:
@dentarg, that was my intention by adding that line, am I missing something?

Reply:
Puma runs on macOS too: https://github.com/puma/puma/blob/a72c68b350e8c48b6c051ee729dd156c6044ca9d/.github/workflows/puma.yml#L22